Specs/DateTime/Korean/DateExtractor.json

```diff
@@ -1040,6 +960,7 @@
   },
   {
     "Input": "너는 지금부터 한 달에 두 번씩 일요일마다 시간 괜찮니? ",
+    "Comment": "Not a proper sentence in Korean and it is not possible to give a better translation of the original case 'Are you available two sundays from now?'",
```

Comment:
In these two cases, please delete the specs that are not a proper sentence.

Reply:
ok, will do